cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/UserApi.java

```diff
@@ -129,15 +130,26 @@
     @POST( "user/verification" )
     Call<Void> requestVerification();
 
+    /**
+     * @deprecated look for {@link #savePaymentMethod(PaymentMethodDTO)}
+     */
+    @Deprecated
     @POST( "user/payment/card" )
     Call<Void> saveCardData( @Body CardDataDTO data );
 
     @POST( "user/payment/sepa" )
     Call<Void> saveSepaData( @Body SepaDataDTO data );
 
+    /**
+     * @deprecated look for {@link #savePaymentMethod(PaymentMethodDTO)} ()}
+     */
+    @Deprecated
     @POST( "user/payment/token" )
     Call<Void> saveTokenData( @Body TokenDataDTO data );
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)